### ISSUE TITLE
No Results Links Should Have a Default Focus State

### ISSUE THREAD

**Issue by rosiegrant** [yext/answers-search-ui#1035]
Per WCAG best practice, the links in the no results module should have a focus state. 

```
.yxt-AlternativeVerticals-suggestion:focus, .yxt-AlternativeVerticals-universalLink:focus {
  color: var(--yxt-color-link-primary);
  text-decoration: underline;
}
```
